### ISSUE TITLE
Thumbnails do not show up, Not a JPEG file: starts with 0xff 0x50

### ISSUE THREAD

**Issue by dschrempf** [Beep6581/RawTherapee#5081]
Directory names including a comma `,` are not loading (thumbnails do not show and CPU usage jumps to 100 percent on all cores). I did not test if this was also the case with other special characters. I am using RawTherapee 5.4 on Arch Linux.

Edit: I noticed that some other directories including a comma do actually work, so this must be another problem...
Edit: On the console, I get the following error:

...
Not a JPEG file: starts with 0xff 0x50
Not a JPEG file: starts with 0xff 0x50
...

But the files open perfectly, e.g., with `gthumb`.

**Comment by dschrempf** [comment]
It turned out, that I had pictures with a JPG ending that were actually PNG images and that RawTherapee does not work with PNG images. A meaningful error message would be nice in this case. Sorry for the noise! Feel free to close the issue in case you do not plan implement this. Thank you!

**Comment by heckflosse** [comment]
@dschrempf 

> that RawTherapee does not work with PNG images

PNG is not enabled by default.
![grafik](https://user-images.githubusercontent.com/13733487/49698848-e263f500-fbc9-11e8-8dea-cfd6ff4d441d.png)

**Comment by Beep6581** [comment]
The message `Not a JPEG file: starts with 0xff 0x50` does not come from RawTherapee but from one of the libraries it uses. As agreed to in IRC, we will try to add the filename to that message for 5.6.

**Comment by Thanatomanic** [comment]
Isn't the issue here that RT actually doesn't recognize a badly named extension? If I call my NEF a JPG, should it give an error, warning or silently interpret the file for what it is?

**Comment by dschrempf** [comment]
Yes, this was the problem. I had PNG files with JPG extension (created by a bad program that adds some "effects", I had not done that myself). RawTherapee tried to open those and failed. After converting them to JPG, everything works.

Thank you for making me aware of the fact that PNG can actually be used in RawTherapee!
